packages/manager/modules/anthos/src/dashboard/host/reinstall/template.html

```diff
@@ -0,0 +1,11 @@
+<oui-modal
+    data-primary-action="$ctrl.reinstallHost()"
```

Comment:
`loading` attribute is missing here and in controller

Reply:
Updated